testing/src/main/scala/sbt/JUnitXmlTestsListener.scala

```diff
@@ -59,7 +60,7 @@ class JUnitXmlTestsListener(val outputDir: String) extends TestsListener {
    * Gathers data for one Test Suite. We map test groups to TestSuites.
    * Each TestSuite gets its own output file.
    */
-  class TestSuite(val name: String) {
+  class TestSuite(val name: String, timestamp: LocalDateTime) {
```

Comment:
Mima failed on Travis CI. It's unlikely that sbt plugins would instantiate this class, but adding another constructor that passes in `LocalDateTime.now()` should be easy to keep the bincompat, so could you plz add it?

Reply:
✅